### DISCUSSION TITLE
Issue setting up Copilot with Visual Studio 17.3.4

### DISCUSSION THREAD

**Discussion by ChrisPorter87** [community/community#34483] [Copilot Conversations]
I'm having a roadblock with Copilot setup with Visual Studio version 17.3.4 and the current Copilot version of 1.46.0.2.  I follow along with the documentation and it states that a device activation code will be saved to the clipboard and that I'll need to paste it somewhere.  When reading a similar post on this forum, it seems as though when you click OK on that box informing you about the activation code, that it should automatically open a tab on the Copilot site which will allow me to paste said code and verify my device.  However, when I hit ok nothing happens, and I'm unable to determine where to paste this code.  Someone have any guidance? Thanks a lot!

**Comment by iamcjwilliams** [comment]
Hello, @ChrisPorter87 Are you referring to the documentation [here](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio)? Here are the steps for login from a clean state:

1. Close Visual Studio 2022.
2. Delete C:\Users\<insert username>\AppData\Local\github-copilot\hosts.json
3. Log out of GitHub in the web browser.
4. Open Visual Studio.

**Reply by ChrisPorter87** [reply]
Yes I am referring to that.  I appreciate your help and response.  Sadly within the local folder I'm not seeing anything for Copilot, and tried to search for the hosts.json file and the search yielded no results.
